src/navigation/checkout/components/RestaurantProfile.js

```diff
@@ -0,0 +1,154 @@
+  },
+  logoWrapperShadow: {
+    elevation: 8,
+    shadowColor: '#00000044',
+    borderRadius: 8,
+    shadowOpacity: 0.25,
+    shadowRadius: 16,
+    shadowOffset: { height: 8, width: 0 },
+  },
+  description: {
+    marginTop: 8,
+  },
+  tags: {
+    display: 'flex',
+    flexDirection: 'row',
+    gap: 8,
+    flexWrap: 'wrap',
+    marginTop: 8,
+  },
+  logo: {
+    borderRadius: 8,
+    aspectRatio: '1',
+    width: '100%',
+  },
+  banner: {
+    aspectRatio: '16/9',
+    width: '100%',
+  },
+  badgesScroll: {
+    width: '100%',
+    overflow: 'hidden',
+  },
+  badgesWrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    gap: 8,
+    left: 0,
+    marginTop: 4,
+    marginBottom: 8,
+    paddingHorizontal: 16,
+  },
+  address: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    alignContent: 'center',
+    // alignSelf: "center",
+    textAlignVertical: 'center',
+    gap: 4,
+  },
+});
+
+function RestaurantProfile({ restaurant, onInfo }) {
+  const backgroundColor = useColorModeValue('#fff', '#201E1E');
+  const stroke = useColorModeValue('#000', '#fff');
+
+  return (
+    <View style={([styles.profile], { backgroundColor })}>
+      <Image
+        style={styles.banner}
+        resizeMode="cover"
+        source={{
+          uri: restaurant.bannerImage
+            ? restaurant.bannerImage
+            : restaurant.image,
+        }}
+        alt="Banner"
+      />
+      <View style={styles.detailsWrapper}>
+        <View style={[styles.logoWrapper, { backgroundColor }]}>
+          <View style={styles.logoWrapperShadow}>
+            <Image
+              style={styles.logo}
+              resizeMode="cover"
+              source={{ uri: restaurant.image }}
+              alt="logo"
+            />
+          </View>
+        </View>
+        {restaurant?.badges?.length >= 1 ? (
+          <ScrollView
+            style={styles.badgesScroll}
+            horizontal={true}
+            showsHorizontalScrollIndicator={true}>
+            <View style={styles.badgesWrapper}>
+              {restaurant.badges.map((badge, i) => (
+                <RestaurantBadge type={badge} key={i} />
+              ))}
+            </View>
+          </ScrollView>
+        ) : null}
+      </View>
+      <View style={styles.content}>
+        <View style={{ display: 'flex', flexDirection: 'row', gap: 4 }}>
+          <TimingBadge restaurant={restaurant} />
+          <Button size="sm" variant="link" onPress={onInfo}>
+            {i18next.t('RESTAURANT_MORE_INFOS')}
+          </Button>
+        </View>
+        <View style={styles.address}>
+          <AddressIcon stroke={stroke} />
+          <Text>{restaurant.address.streetAddress}</Text>
+        </View>
+        {restaurant.description ? (
+          <View style={styles.description}>
+            <Text>{restaurant.description}</Text>
+          </View>
+        ) : null}
+        {restaurant.tags?.length > 0 ? (
```

Comment:
In the designs tags are displayed together with badges:

![Screenshot 2024-03-14 at 15 20 14](https://github.com/coopcycle/coopcycle-app/assets/70273239/0b997af9-6afd-495c-a801-41f84a5e2552)

Reply:
It's going to be complicated, some restaurants have a lot of tags

Reply:
On the web it's just a flexbox with wrap:

```
.badges-and-tags {
    display: flex;
    gap: 8px;
    flex-wrap: wrap;
  }
```

Reply:
I mean, if we grouped tags and badges together and displayed them at the top of the profile, it wouldn't make much sense because tags are usually complementary information that's displayed at the bottom. Especially when the restaurant has twenty or so tags, the design wouldn't make much sense

Conversely, displaying tags and badges together at the bottom isn't a good idea either, as badges such as "exclusive" won't have enough visibility for what they represent

In my opinion, these are two pieces of information with different levels of visibility, which is why I've separated them

Reply:
Okay, that makes sense, but let's check with Vincent and implement it the same on both web and app

Reply:
Yes, same for the alert and all the other little things